src/domain/hanke/edit/components/Haittojenhallintasuunnitelma.tsx

```diff
@@ -0,0 +1,110 @@
+import useFieldArrayWithStateUpdate from '../../../../common/hooks/useFieldArrayWithStateUpdate';
+import HankealueMap from '../../../map/components/HankkeenHaittojenhallintasuunnitelma/HankealueMap';
+import VectorSource from 'ol/source/Vector';
+import useAddressCoordinate from '../../../map/hooks/useAddressCoordinate';
+
+type Props = {
+  hanke: HankeData;
+  index: number;
+};
+
+const Haittojenhallintasuunnitelma: React.FC<Props> = ({ hanke, index }) => {
+  const { t } = useTranslation();
+  const { fields: hankealueet } = useFieldArrayWithStateUpdate<HankeDataFormState, 'alueet'>({
+    name: FORMFIELD.HANKEALUEET,
+  });
+  const hankealue = hankealueet[index];
+  const haittojenhallintatyypit = sortedLiikenneHaittojenhallintatyyppi(
+    hankealue.tormaystarkasteluTulos as HankeIndexData,
+  );
+  const [drawSource] = useState<VectorSource>(new VectorSource());
+  const addressCoordinate = useAddressCoordinate(hanke.tyomaaKatuosoite);
+
+  return (
+    <div>
+      <Box mt="var(--spacing-m)" mb="var(--spacing-xs)" fontWeight="bold">
+        {t('hankeForm:haittojenHallintaForm:subHeaderPlan')}
+      </Box>
+      <Box mb="var(--spacing-m)" ml="var(--spacing-l)">
+        <Trans i18nKey="hankeForm:haittojenHallintaForm:instructionsGeneral">
+          <ul>
+            <li>Varmista jalankulun turvallisuus ja esteettömyys</li>
+            <li>Varmista kulkuyhteydet kiinteistöihin</li>
+            <li>Tee yhteensovitus muiden hankkeiden ja töiden sekä niiden kiertoreittien kanssa</li>
+            <li>
+              Ilmoita aina katujen sulkemisesta liikenteeltä Pelastuslaitokselle ja Poliisille
+            </li>
+            <li>Tarkista aina, onko hankkeesi tai työsi erikoiskuljetusten reitillä</li>
+            <li>
+              Tarkista vaikutukset olevaan liikennevalo-ohjaukseen (kaistajärjestelyt, ilmaisimet,
+              valojen toimivuus)
+            </li>
+            <li>Poikkeavat työajat</li>
+          </ul>
+        </Trans>
+      </Box>
+      <div key={HAITTOJENHALLINTATYYPPI.YLEINEN} className="formWpr">
+        <TextArea
+          name={`${FORMFIELD.HANKEALUEET}.${index}.haittojenhallintasuunnitelma.${HAITTOJENHALLINTATYYPPI.YLEINEN}`}
+          label={t(
+            `hankeForm:labels:haittojenhallintasuunnitelma:${HAITTOJENHALLINTATYYPPI.YLEINEN}`,
+          )}
+          testId={`${FORMFIELD.HANKEALUEET}.${index}.haittojenhallintasuunnitelma.${HAITTOJENHALLINTATYYPPI.YLEINEN}`}
+          required={true}
+          helperText={t('hankeForm:haittojenHallintaForm:helperText')}
+        />
+      </div>
+      <Box mb="var(--spacing-m)">
+        {t('hankeForm:haittojenHallintaForm:subHeaderTrafficNuisanceIndex')}
+      </Box>
+      {haittojenhallintatyypit.map((haitta) => (
+        <div key={haitta} className="formWpr">
+          <Box className="nuisanceType">
+            {t(`hankeForm:haittojenHallintaForm:nuisanceType:${haitta}`)}
+          </Box>
+          <Box mb="var(--spacing-m)">
```

Comment:
According to Figma there should be a margin between heading and map

Reply:
Added `mt="var(--spacing-m)"` for the map box.